plugin-dev/Config/FilterPlugin.ini

```diff
@@ -7,5 +7,10 @@
 ;    /Extras/...
 ;    /Binaries/ThirdParty/*.dll
 /Scripts/*
-/LinuxIntermediate/*
```

Comment:
Why doesn't this work?

Reply:
Apparently the `*` wildcard doesn't prevent UE Marketplace from obfuscating subdirectories in `LinuxIntermediate` folder during the publishing so I assume that specifying those explicitly should help